bootstrap/pesticide_seed2.py

```
@@ -0,0 +1,1233 @@
+                    "PostgreSQL 서버 연결 재시도 초과: "
+                    f"{host}:{port}, attempts={max_retries}, last_error={exc}"
+                ) from exc
+            log(
+                "PostgreSQL 서버 응답 대기 중: "
+                f"{host}:{port}, attempt={attempt}/{max_retries}, error={exc}. "
+                f"{int(POSTGRES_RETRY_INTERVAL_SECONDS)}초 후 재시도합니다."
+            )
+            time.sleep(POSTGRES_RETRY_INTERVAL_SECONDS)
+
+
+def rebuild_tables(engine: Any) -> None:
```

Ruff (0.15.12)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `engine`

(ANN401)